test/components/portal.spec.tsx

```diff
@@ -1,36 +1,38 @@
 import * as React from 'react';
 import * as ReactDOM from 'react-dom';
-import {ClientRenderer, expect, selectDom, waitFor} from 'test-drive-react';
+import {ClientRenderer, expect, waitFor} from 'test-drive-react';
 import {Portal} from '../../src';
-
-const portalId = 'PORTAL';
+import {PortalTestDriver} from '../../test-kit/components/portal-driver';
 
 describe('<Portal />', function() {
     const clientRenderer = new ClientRenderer();
-    const bodySelect = selectDom(document.body);
 
     afterEach(function() {clientRenderer.cleanup(); });
 
     it('displays the portal and renders its children', async function() {
-        clientRenderer.render(
+        const {result} = clientRenderer.render(
             <Portal>
                 <span data-automation-id="SPAN">Portal Body</span>
             </Portal>);
 
+        const driver = new PortalTestDriver(result as Portal);
```

Comment:
rename `driver` to `portal` for better readability

Reply:
done